docs/user-guide/src/ipam/ipam_install.md

```diff
@@ -0,0 +1,56 @@
+# Installing IPAM as Deployment
+
+This section will show how  IMAP can be installed as a deployment in a cluster. We will discuss the deployment worklow of IPAM.
+
+## Deploying controllers
+
+CAPI and IPAM controllers need to be deployed at the begining. The IPAM controller has a dependency on Cluster API *Cluster* objects. CAPI CRDs and controllers must be deployed and the cluster objects should exist for
+successful deployments.
+
+## Deployment
+
+User can create the **IPPool** object independently. It will wait for its cluster to exist before reconciling. If an user wants to  deploy **IPAddress** objects manually, they should be deployed before any claims. It is highly recommended to use the *preAllocations* field itself or have the reconciliation paused.
```

Comment:
```suggestion
The user can create the **IPPool** object independently. It will wait for its cluster to exist before reconciling. If the user wants to  create **IPAddress** objects manually, they should be created before any claims. It is highly recommended to use the *preAllocations* field itself or have the reconciliation paused.
```